tests/utils/reporting.py

```diff
@@ -24,6 +25,13 @@
 T = TypeVar("T", bound=Callable[..., Any])
 
 
+class CustomJSONEncoder(json.JSONEncoder):
```

Comment:
Changed because `safeMint2` and `safeTransfer2` takes a byte argument, and reporting is dumping arguments via json.dumps.

Without this change, the test runner throws an error that bytes is not json serializable.